packages/easy-coding-standard/build/target-repository/composer.json

```diff
@@ -16,5 +21,8 @@
     "conflict": {
         "squizlabs/php_codesniffer": "<3.6",
         "friendsofphp/php-cs-fixer": "<3.0"
+    },
+    "config": {
+        "vendor-dir": "vendor-tests"
     }
```

Comment:
Not needed

Reply:
The vendor directory in ecs repository contains prefixed dependencies of ecs. It could easily cause unexpected issues if you try to install more packages into that directory. Also makes it difficult to work with the repo since those new filed would appear among git changes. I strongly recommend keeping the directories separate.

Reply:
This is not needed, you're testing something else here

Reply:
Err... okay you got me confused here... first you added a comment that doesn't address my concerns at all, claiming it's not necessary without a counter argument but then you marked the thread as resolved... should I leave it as is then?